airflow/utils/docs.py

```diff
@@ -39,17 +39,25 @@ def get_docs_url(page: str | None = None) -> str:
     return result
 
 
+def get_project_url_from_metadata(provider_name: str):
+    """Returns the Project-URL from metadata."""
```

Comment:
```suggestion
    """Return the Project-URL from metadata."""
```
I guess that should fix the CI.

Reply:
Thank you, just pushed a commit